### DISCUSSION TITLE
reading jsonb data

### DISCUSSION THREAD

**Discussion by LisandroFilloy** [sfu-db/connector-x#311] [Q&A]
Hello guys, I arrived here after using polars to read data from a postgresql DB table. Polars relies on connectorx to read data from dbs. I am not being able to read jsonb data, as it seems connectorx is not capable of reading this. Any idea on how I could address this? 
Error message: 
![Screen Shot 2022-07-07 at 12 46 39](https://user-images.githubusercontent.com/63921959/177816169-cda594d3-7f6c-4a40-9ccc-cd9574522a7c.png)
Is there any idea to implement this feature in the future?

**Comment by wangxiaoying** [comment]
Hi @LisandroFilloy , can you try whether setting `return_type=arrow2` resolves the issue?
